Sources/Utility/PkgConfig.swift

```diff
@@ -59,9 +59,9 @@ public struct PkgConfig {
     /// - fileSystem: The file system to use, defaults to local file system.
     ///
     /// - throws: PkgConfigError
-    public init(name: String, fileSystem: FileSystem = localFileSystem) throws {
+    public init(name: String, additionalSearchPaths: [AbsolutePath] = [], fileSystem: FileSystem = localFileSystem) throws {
```

Comment:
Can you documentation for this above the init method stating the priority in which this will be searched